src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/api/GetAllocationResponse.kt

```diff
@@ -46,4 +46,7 @@ data class GetAllocationResponse(
 
   @Schema(description = "Rules for creating schedules - days and times")
   val scheduleRules: List<ScheduleRulesResponse> = listOf(),
+
+  @Schema(description = "The start date of the activity", example = "2023-03-12")
+  val activityStartDate: LocalDate,
```

Comment:
We're going to start allowing migration of activities that start in the future. When this happens we'll need to make sure that the allocation doesn't start before the activity or this will break DPS - hence getting activity start date in the allocation response.